drivers/net/ethernet/qualcomm/qca_spi.c

```diff
@@ -581,6 +581,11 @@ qcaspi_spi_thread(void *data)
 	netdev_info(qca->net_dev, "SPI thread created\n");
```

Comment:
Just couple of stuff regarding the commit message:

- "This is a big issue because it allows userspace to prevent from restarting the SPI thread..." So the issue is that the userspace is prevented from restarting the thread or that the userspace itself is not allowing the thread to restart?
- "The open/close ops could be realized just by park/unpark" should be parking/unparking.. but I see that you are referring the kernel thread functions so maybe but it in single quotes

Reply:
1. The problem is that these functions can be called by user context. In case the user hit Strg+C the signal HUP would be send to the SPI thread of the driver which is unintended.
2. Yes, i'm refering to the kernel thread functions.

Reply:
Please have a look

Reply:
looks good